scripts/add_map_plugin.py

```diff
@@ -0,0 +1,75 @@
+# -------------------------------------------------------------------------
+
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -------------------------------------------------------------------------
+
+import sys
+import os
+import subprocess
+import argparse
+import json
+
+
+def process_args(args, default_args):
+	'''
```

Comment:
Are indents tabs here? We should follow Python's [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation), which means 4 spaces for indentation.

Reply:
Oh good point !